man/docker-run.1.md

```diff
@@ -330,6 +331,13 @@ redirection on the host system.
 **--init-path**=""
    Path to the docker-init binary
 
+**--hugetlb-limit**=[]
+  Set the container huge page limit (format: [size:]<limit>, e.g. --hugetlb-limit 2MB:32MB)
+
+  Constrains the hugetlb page limit to a container. Hugetlb page size must be legal, it will be
+replaced with host default huge page size if not set. Limit should be times of page size,
```

Comment:
"Limit should be times of page size" -> "Limit should be a multiple of the huge page size"